Backend/dev/migrations/dynamic-offer-driver-app/0155-add-s3_media_file_url_pattern.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE atlas_driver_offer_bpp.transporter_config
+ADD COLUMN s3_media_file_url_pattern Text NOT NULL DEFAULT 'http://localhost:8016/ui/<DOMAIN>/s3media?filePath=<FILE_PATH>';
```

Comment:
Host can also be made a pram and be fetched from dhall in code, but yeah, this is fine and simple too. Just that the number of queries will be different in prod/master, we'll have to manually paste the host here during release.

Reply:
+1

Reply:
so how will I do this ?